app/src/main/res/drawable/widget_balances_background.xml

```diff
@@ -1,4 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <shape xmlns:android="http://schemas.android.com/apk/res/android">
-    <solid android:color="@android:color/transparent" />
+
+    <solid android:color="@color/ic_launcher_background" />
+
+    <corners android:radius="8dp" />
```

Comment:
Why are doing this? The background should be cropped/ shaped by the system not use